shared/src/main/kotlin/org/javacs/kt/classpath/ClassPathResolver.kt

```diff
@@ -26,11 +28,13 @@ interface ClassPathResolver {
             emptySet<Path>()
         }
 
+    fun fetchClasspathWithSources(): Set<ClassPathEntry> = classpath
```

Comment:
Is there a specific reason for making this a function? Otherwise I would suggest making it a computed property, like the other classpath-related members:

```kotlin
val classpathWithSources: Set<ClassPathEntry>
    get() = classpath
```